traits/tests/test_configure_traits.py

```diff
@@ -0,0 +1,92 @@
+# Copyright (c) 2019 by Enthought, Inc.
+# All rights reserved.
+
+try:
+    # Python 3: mock in the standard library.
+    import unittest.mock as mock
+except ImportError:
+    # Python 2: need to use 3rd-party mock.
+    import mock
+import os
+import shutil
+import tempfile
+import unittest
+
+import six
+
+from traits.api import HasTraits, Int
+from traits.testing.optional_dependencies import requires_traitsui
+
+
+class Model(HasTraits):
+    count = Int
+
+
+@requires_traitsui
+class TestConfigureTraits(unittest.TestCase):
+    def setUp(self):
+        import traitsui.api
```

Comment:
any specific reason why you want to import inside the `setUp` instead of just updating the `...optional_dependencies` import?

Reply:
The `optional_dependencies` only has `traitsui` available, so you'd then have to test whether `traitsui` is `None` and if not, do another import. (`traitsui.api` is not immediately available from an `import traitsui`) It's less complicated this way.